src/views/List.jsx

```diff
@@ -1,8 +1,19 @@
-import { ListItem } from '../components';
 import { useState } from 'react';
 import { NavLink } from 'react-router-dom';
+import { comparePurchaseUrgency } from '../api/firebase';
+import { ListItemGroup } from '../components/ListItemGroup';
+
+const itemUrgencyCategory = {
+	0: 'Overdue',
+	1: 'Soon',
+	2: 'Kind of Soon',
+	3: 'Not Soon',
+	4: 'Inactive',
+	5: 'Purchased',
+};
```

Comment:
Typically in an object if your keys are numbers starting from zero, you can convert this to an array (since `itemUrgencyCategory[0]` would be the syntax you use either way)

Reply:
@jendevelops I think I remember you saying something about taking this a step further and making an array of objects that include the category label _and_ the number of days associated with it. I was exploring this and ultimately decided not to refactor AddListItem to utilize it, because I really like how @a-cole55 and @mariahschock used numDaysInEstimate.